week1/assignment/todo-tan/todo.rb

```diff
@@ -0,0 +1,167 @@
+require_relative "list"
+
+class Todo
+  def initialize(filename = "todo.md")
+    @filename = filename
+    @list = List.new("Today")
+    @trash_list = List.new("Trash")
+    @last_request = {}
+  end
+
+  def list
+    @list
+  end
+
+  def load_data
+    file = File.read(@filename)
+    lines = file.split("\n")
+    lines.each do |line|
+      @list.add(Item.new_from_line(line))
+    end
+  end
+
+  def save(filename = "todo.md")
+    File.write(filename, @list.to_string)
+  end
```

Comment:
technically, you probably want to save to `File.write(@filename, ...)` because `@filename` is already set inside this class